README.md

```diff
@@ -0,0 +1,48 @@
+# Apple Landing page clone
+
+> this is a clone of the Apple 2014 landing page. The goal was to make it look as similar as possible.
+
```

Comment:
Improve your Readme file. It should be descriptive of the project. You are using the correct template but now you need to fill it with information about your project. You can add details about what you did in this project, why you did it, how you did it...Check this [link](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/) for more info.